### ISSUE TITLE
Merge Cells with Wrap Text Breaks Height

### ISSUE THREAD

**Issue by HunterMeyer** [randym/axlsx#543]
I'm attempting to merge two cells in a row. I have the row style set to `alignment: { wrap_text: true }`.
My expectation is that the height of the row would _stretch_ vertically to fit the contents of the cell (which Excel does when merging cells).
However, the cell height remains fixed and causes the cell contents to be unreadable.

### Here's a screenshot
Notice in the highlighted section you only see the bottom portion of text.
![screen shot 2017-08-16 at 6 48 50 pm](https://user-images.githubusercontent.com/4690516/29388679-0fe753ae-82b4-11e7-846d-b321bb468795.png)

### Without merging
The text wraps correctly and the height of the cell is adapted.
![screen shot 2017-08-16 at 6 49 24 pm](https://user-images.githubusercontent.com/4690516/29388685-15cfdf84-82b4-11e7-8d91-f8f5bd72a55c.png)

### Basically what I'm doing
```ruby
excel  = Axlsx::Package.new
wb     = excel.workbook
style  = wb.styles.add_style(alignment: { wrap_text: true })
wb.add_worksheet(name: 'Some File') do |sheet|
  array_of_text.each_with_index do |text, row_num|
    row_num += 1
    sheet.add_row [text], style: style
    sheet.merge_cells("A#{row_num}:B#{row_num}")
    sheet.column_widths 30
  end
end
```

Am I doing something wrong or is this a bug?

**Comment by nachokb** [comment]
I'm having this issue as well…

**Comment by nachokb** [comment]
This seems to be an Excel limitation (!)

ref [Microsoft forum](https://answers.microsoft.com/en-us/office/forum/office_2007-excel/autofit-row-height-doesnt-work/ca2a86b8-585a-46fd-a8a7-377e76ac5528?msgId=38709b42-daa4-4739-ae8c-ab8dcf2e2f0c) ([source](https://stackoverflow.com/questions/19624791/how-do-i-auto-fit-a-row-height-after-merging-cells#comment29163059_19624791))

**Comment by Manidavid** [comment]
Am still facing this issue, any solution ?
